src/test.rs

```diff
@@ -270,11 +276,11 @@ pub async fn parent_contact(handle: &Handle, child: &ChildHandle) -> ParentCaCon
     .await
     {
         ApiResponse::ParentCaContact(contact) => contact,
-        _ => panic!("Expected RFC8183 parent response"),
+        _ => panic!("Expected idexchange:: parent response"),
```

Comment:
This probably wasn't what you intended, right?

Reply:
fixed